tangelo/linq/circuit.py

```diff
@@ -126,6 +127,11 @@ def counts(self):
         """Return the counts for all gate types included in the circuit."""
         return self._gate_counts
 
+    @property
+    def counts_n_qubit(self):
+        """Return the counts for all gate types included in the circuit."""
```

Comment:
You just copy-pasted the docstring from L127. Let's turn this into something better:

`"""Return the number of 1-qubit gates, 2-qubit gates, and so on."""`